build.gradle

```diff
@@ -17,7 +17,29 @@ plugins {
     id 'org.springframework.boot' version '2.6.6'
 }
 
-version = '5.1.1-SNAPSHOT'
+final def externalRepoHost = "https://sig-repo.synopsys.com"
+final def internalRepoHost = System.getenv('SNPS_INTERNAL_ARTIFACTORY')
+
+repositories {
+    println "Checking if environment property SNPS_INTERNAL_ARTIFACTORY is configured: ${internalRepoHost}"
```

Comment:
Are we actually "checking" something?

Reply:
Yes, lines 26, 29 and 32 are the checks.